paraclear/README.md

```diff
@@ -0,0 +1,73 @@
+# Paraclear Account Value Checker
+
+This repository contains a Python script designed to interact with a Starkware-based smart contract to retrieve account value information, including perpetual asset balances. Specifically, it retrieves margin health checks data for an account on Paradex
+
+## Features
+
+- Retrieves account value
+- Checks account health (Initial Margin and Maintenance Margin)
+- Lists perpetual balances for available markets
```

Comment:
I wouldn't exactly call these features.  Let's merge this with the first section and then we also need to go into more detail on where we're getting info etc.

If someone wanted to build something similar in another language how would they do it?  Where is this info coming from?  Is there more info that they could potentially pull?  What does the information that we pull corresond to in regards to the UI ?